From e179557771c97238ebf296cc8bb2a2228b3a2e8b Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Mon, 15 May 2017 14:23:09 -0400
Subject: [PATCH 1663/2094] drm/amd/display: Remove duplicate entry from
 log_mask

As pointed out by kbuild test robot and Julia Lawall.

CC: Julia Lawall <julia.lawall@lip6.fr>
CC: Alex Deucher <alexander.deucher@amd.com>

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
---
 drivers/gpu/drm/amd/display/dc/basics/logger.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/display/dc/basics/logger.c b/drivers/gpu/drm/amd/display/dc/basics/logger.c
index 8f44973..4be8370 100644
--- a/drivers/gpu/drm/amd/display/dc/basics/logger.c
+++ b/drivers/gpu/drm/amd/display/dc/basics/logger.c
@@ -81,7 +81,6 @@ static const struct dc_log_type_info log_type_info_tbl[] = {
 		(1 << LOG_SYNC) | \
 		(1 << LOG_BANDWIDTH_VALIDATION) | \
 		(1 << LOG_MST) | \
-		(1 << LOG_DETECTION_EDID_PARSER) | \
 		(1 << LOG_DETECTION_DP_CAPS) | \
 		(1 << LOG_BACKLIGHT)) | \
 		(1 << LOG_I2C_AUX) | \
-- 
2.7.4

